source/users.rst

```diff
@@ -0,0 +1,114 @@
+-----
+Users 
+-----
+
+There are 2 types of users:
+
+1. Those that create / produce data
+2. Those that consume data 
+
+One can both consume data and produce it, or just have one of these roles.
```

Comment:
I have a feeling that if we relate to the table, a user of one level is usually the producer at the next level. The ones calibrating data are users of level-0 and so on. Do we really have a situation where someone only has one role, then?

Reply:
Well, I guess it the producer is a sensor... then yes?

Reply:
yes, I think we do... I specified "at the start/end of the production lines" or something like that